Bottom: de87f5a87dd536025f4ce22a3bad2600d4350081
Top:    fdd78452d7e02d706c88b4821b9ff48bc33218cb
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-05-02 13:47:51 -0700

lib: Rename configure.in to configure.ac

Update to latest autotools naming.

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/configure.in b/configure.ac
similarity index 100%
rename from configure.in
rename to configure.ac
